book/src/hello_world.md

```diff
@@ -76,4 +76,6 @@ The proof can be verified by anyone via:
 powdr verify hello_world.asm --field bn254 --backend halo2 --vkey "vkey.bin" --params "params.bin" --proof "hello_world_proof.bin"
```

Comment:
You mean inconsistency here? There is no path to `hello_world.asm`?

I agree this is inconsistent, but I think it is better to add the path to this one than to remove from the other, so that the user can just copy and paste and it should work.